### ISSUE TITLE
[help/explanation] Multiple streams to m3u without buffer

### ISSUE THREAD

**Issue by jeremyschiemann** [xteve-project/xTeVe#287]
Hi, i have a vbox tv tuner which has 2 tuners. Since it doesnt work nicely with plex im using xteve (it can't play some channels for whatever reason).

I configured xteve to match the 2 tuners and added the playlist from the vbox. Without a buffer selected i can only stream one channel, trying to stream another one crashes every stream. (Adding the vbox directly to plex allows 2 different streams)

When selecting xteve as buffer and then configure the playlist to allow 2 streams (in addition to make xteve have 2 tuners) it works as intended.

However i dont quite understand why multiple streams to the same playlist isnt possible when, like in my case, there are 2 tuners behind that playlist. Shouldn't xteve just forward the correct link and the not be involved anymore?

Tl;dr: why cant I use multiple streams to a playlist without a buffer?

**Comment by MCTyler** [comment]
Did you contact VBox? Their advertisement says you can.

**Comment by jeremyschiemann** [comment]
it does work if I add the vbox directly to plex and it also does work if I use xteve with a buffer selected. So the vbox works correctly.

The only setup that doesnt work is if I use xteve without a buffer. I mean it works perfectly fine with the buffer selected, I'm just curious why that is, the logs dont say anything unusual either.
